pkg/operator/webhookconfig.go

```diff
@@ -72,7 +72,7 @@ func (w *WebhookConfigUpdater) Start(stopCh <-chan struct{}) error {
 	klog.Infof("Webhook configuration status: %s", op)
 
 	// Block until the stop channel is closed.
-	<-stopCh
+	stop.Done()
```

Comment:
This is not equivalent to previous code sample, should be 
```go
<-stop.Done()
```

Reply:
thanks Danil, i'll fix it up